cmd/cli/plugin-admin/builder/command_test.go

```diff
@@ -28,48 +26,34 @@ func Test_BuilderInitAndAddPlugin(t *testing.T) {
 	var stdout, stderr bytes.Buffer
 
 	// Assert dry-run does not create a repo
-	expected := "module testrepo"
 	args := []string{"testrepo", "--dry-run", "--repo-type", "github"}
 	cmd := NewInitCmd()
 	cmd.SetArgs(args)
 	cmd.SetOut(&stdout)
 	cmd.SetErr(&stderr)
 	err = cmd.Execute()
 	assert.Nil(err)
-	assert.Contains(expected, stdout.String())
```

Comment:
If we've removed all assertions from this code is it still testing anything?

Are there more appropriate assertions that should be used now?

Reply:
I'm generally against matching string output in unit tests as that's usually a poor indicator that is prone errors due to changes. Which IIRC was the case here that led me to changing these.

The main thing we care about is on the previous line in asserting that running this command does not return an error.